common-api/src/main/java/org/pmiops/workbench/config/SurveyModulesConfig.java

```diff
@@ -0,0 +1,30 @@
+package org.pmiops.workbench.config;
+
+import java.util.List;
+
+public class SurveyModulesConfig {
```

Comment:
I wrote this when I was experimenting with having survey module info represented as JSON configuration instead of rows in dedicated tables. I abandoned that in this PR (mostly because I want to be able to update the counts for survey modules using SQL.) Anyway, we can revisit this later when we have a better story around PPI; for now, deleting this.